challenges/objects-arrays.js

```diff
@@ -51,20 +78,48 @@ const graduates = [
 /* Request 1: Create a new array called universities that contains all the universities in the graduates array. This will be an array of strings.
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
+
+// we are going inside to extract something, build a loop related to array length
+//.sort() sorts in into alphabetic order
+// .push() adds 1 more element to the array and returns the new length
+
 const universities = [];
-console.log(universities);
+
+for (i=0; i<graduates.length; i++) {
+  universities.push(graduates[i].university)
+}
+console.log(universities.sort());
```

Comment:
be careful about returning a chained method usually it will work out fine but I would rather see you sorting it into a sortedArray and returning that sorted array

Reply:
I remember you clearly stating this.  My bad.... I could do it this way:
for (i=0; i<graduates.length; i++) {
  universities.push(graduates[i].university)
}
universities.sort();

console.log(universities)

Reply:
Even better...I could also do: 
const = uniSort = universities.sort()
console.log(uniSort)